src/common/utility.ts

```diff
@@ -566,6 +566,10 @@ export class Utility {
         if (name.match(/[^a-zA-Z0-9\_]/)) {
             return "The name must contain only alphanumeric characters or the symbol _";
         }
+        if (name.match(/^\d/)) {

```

Comment:
instead of do regex match two times. Can we just match with the legal pattern once?

Reply:
Yes, however, if we use one pattern, the error message maybe not accurate

Reply:
How about error message: "The name cannot start with number and must contain  only alphanumeric characters or the symbol _"

Reply:
Thanks, according to the schema, the validate regex would be /^[a-zA-Z][a-zA-Z0-9_.-]*$/